modules/ovn-kubernetes-architecture-con.adoc

```diff
@@ -24,4 +24,4 @@ It translates the logical network configuration in terms of conventional network
 
 The OVN southbound database has physical and logical representations of the network and binding tables that link them together. It contains the chassis information of the node and other constructs like remote transit switch ports that are required to connect to the other nodes in the cluster. The OVN southbound database also contains all the logic flows. The logic flows are shared with the `ovn-controller` process that runs on each node and the `ovn-controller` turns those into `OpenFlow` rules to program `Open vSwitch`(OVS).
 
-The Kubernetes control plane nodes each contain an `ovnkube-control-plane` pod which does the central IP address management (IPAM) allocation for each node in the cluster. At any given time a single `ovnkube-control-plane` pod is the leader.
+The Kubernetes control plane nodes contain two `ovnkube-control-plane` pods on separate nodes, which does the central IP address management (IPAM) allocation for each node in the cluster. At any given time, a single `ovnkube-control-plane` pod is the leader.
```

Comment:
Somewhat out of scope so up to you, but I think "does" here reads a bit awkward and might not translate as well:
```suggestion
The Kubernetes control plane nodes contain two `ovnkube-control-plane` pods on separate nodes, which performs the central IP address management (IPAM) allocation for each node in the cluster. At any given time, a single `ovnkube-control-plane` pod is the leader.
```

It might not make that sentence any clearer, or it might change the accuracy, so I'll leave it up to you.